tools/Google.Cloud.Tools.SbomGenerator/NuspecPackage.cs

```diff
@@ -45,4 +52,20 @@ public class NuspecMetadata
         [XmlElement("license")]
```

Comment:
nit: In a separate PR, maybe remove these as I believe they are not being used anymore?

Reply:
Done in this one.